__tests__/api.test.js

```diff
@@ -0,0 +1,39 @@
+const request = require("supertest");
+const app = require("../api/app");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const db = require("../db/connection");
+const categories = require("../db/data/test-data/categories");
+
+afterAll(() => {
+  return db.end();
+});
+beforeEach(() => {
+  return seed(testData);
+});
+
+describe("/api", () => {
+  test("GET - Status: 200 - returns an object with message 'all ok'", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(response.body).toEqual({ message: "all ok" });
```

Comment:
Ensure the response is in the format specified on the ticket: {categories: [...]}